### ISSUE TITLE
urls should have title attribute

### ISSUE THREAD

**Issue by kcrisman** [rbeezer/mathbook#22]
There is no tooltip for the `<url href="...">text</url>` tag.  That is against good accessibility guidelines as well as normal `<a>` tag practice.  At least one should be able to do so, but it doesn't seem to work.

**Comment by rbeezer** [comment]
Good suggestion/enhancement.  I'll be getting to these once AIMS concludes.
